OpenSim/Simulation/Wrap/WrapSphereObst.cpp

```diff
@@ -62,7 +62,7 @@ WrapSphereObst::~WrapSphereObst()
 */
 void WrapSphereObst::constructProperties()
 {
-    constructProperty_radius(-1.0);
```

Comment:
+1. It would be backwards-incompatible to change the default value for any properties.

Reply:
As discussed in a previous PR, change to a valid value seems to be the targeted avenue now since this doesn't much negatively affects backward compatibility while more fitting API 4.0